explainability-core/src/main/java/org/kie/trustyai/metrics/drift/fouriermmd/FourierMMD.java

```diff
@@ -0,0 +1,477 @@
+package org.kie.trustyai.metrics.drift.fouriermmd;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.function.Function;
+
+import org.apache.commons.math3.distribution.NormalDistribution;
+import org.apache.commons.math3.linear.Array2DRowRealMatrix;
+import org.apache.commons.math3.random.JDKRandomGenerator;
+import org.apache.commons.math3.random.RandomGenerator;
```

Comment:
Please use `java.util.Random;` unless specifically needing apache functions

Reply:
changed to java.util.Random